src/legacy/ui/public/field_editor/field_editor.js

```diff
@@ -331,6 +335,57 @@ export class FieldEditorComponent extends PureComponent {
     );
   }
 
+  /**
+   * renders a warning and a table of conflicting indices
+   * in case there are indices with different types
+   */
+  renderTypeConflict() {
+    const { intl } = this.props;
+    const { field = {} } = this.state;
+    if (!field.conflictDescriptions || !typeof field.conflictDescriptions === 'object') {
+      return null;
+    }
+
+    const columns = [{
+      field: 'type',
+      name: intl.formatMessage({ id: 'common.ui.fieldEditor.typeLabel', defaultMessage: 'Type' }),
+      width: '100px',
+    }, {
+      field: 'indices',
+      name: intl.formatMessage({ id: 'common.ui.fieldEditor.indexNameLabel', defaultMessage: 'Index names' })
+    }];
+
+    const items = Object
+      .entries(field.conflictDescriptions)
+      .map(([type, indices]) => ({
+        type,
+        indices: Array.isArray(indices) ? indices.join(', ') : ''
```

Comment:
The case where `indices` is not an array, is that a meaningful state? If its not, and its something we should account for, maybe we should show some sort of error text.

Reply:
didn't analyse, if this state is possible, just wanted to rule out an exception, what kind of error message to you suggest

Reply:
Two options, I'll let you chose -

1. Don't check for array, just let it fail. Its the 'fail fast' philosophy.

2. Maybe provide `Index names unavailable` instead of the comma separated index name list...but it would be better to tell the user how to resolve the situation. I'm not sure how we'd resolve this.

Reply:
I've chosen number 2, that's better than an exception, and it's highly unlikely anybody runs into this situation, but if, it's easier to debug